labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="desktop-stylesheet.css">
+  <link rel="stylesheet" href="stylesheet.css">
+  <!--
+        Author: Vikram Badarinath
+        Created: 01/23/2018
+      -->
+  <title>
+    chirpus inc.
+  </title>
+</head>
+
+<body>
+  <div class="menu">
+    <div class="title">
+      chirpus inc.
+    </div>
+    <div class="navbar">
+      <nav class="tab">
```

Comment:
Ok, ok, some confusion here. `nav` is the element you should use for the *whole navbar*, not one element of the navigation.  So `<div class="navbar">` should be `<nav>` and `<nav class="tab"><a...` should be `<a class="tab">...`